support-frontend/assets/components/headers/header/header.scss

```diff
@@ -213,3 +219,103 @@ $header-links-min-width: gu-span(12);
     }
   }
 }
+
+.component-header--display-checkout {
+  .component-header__wrapper {
+    max-width: 50rem;
```

Comment:
try to use `gu-span(*)` here rather than a fixed pixel size

Reply:
Here as well.